pkg/controllers/management/auth/project_cluster_handler.go

```diff
@@ -460,21 +460,12 @@ func (m *mgr) addRTAnnotation(obj runtime.Object, context string) (runtime.Objec
 
 	annoMap := make(map[string][]string)
 
-	switch context {
-	case "project":
-		for _, role := range rt {
-			if role.ProjectCreatorDefault {
-				annoMap["required"] = append(annoMap["required"], role.Name)
-			}
-		}
-	case "cluster":
-		for _, role := range rt {
-			if role.ClusterCreatorDefault {
-				annoMap["required"] = append(annoMap["required"], role.Name)
-			}
+	for _, role := range rt {
```

Comment:
dont understand this set of changes. why did you drop the context stuff and are only checking for cluster creator default?